odonto/odonto_submissions/exceptions.py

```diff
@@ -0,0 +1,2 @@
+class MessageSentException(Exception):
```

Comment:
Why is there also one of these in models ? 
And how come it doesn't inherit from an Error class?